CRM/Financial/BAO/FinancialAccount.php

```diff
@@ -439,16 +439,20 @@ public static function checkFinancialTypeHasDeferred($params, $contributionID =
    * @return array of Deferred Financial Account
    *
    */
-  public static function getAllDeferredFinancialAccount() {
-    $query = "SELECT cfa.id, cfa.name FROM civicrm_entity_financial_account ce
+  public static function getAllDeferredFinancialAccount($includeCode = FALSE) {
```

Comment:
grep -irn 'getAllDeferredFinancialAccount' . says it used at only one place i.e in Deferred Revenue report. Can we exclude this $includeCode